### PR TITLE
add new hydras

### PR DESCRIPTION
**Pull request by hulxv**
# What's new?

### Add more hydras 
I added some useful hydras for these tools:
  - [NPM](https://www.npmjs.com/) (Package manager for nodejs)
  - [Yarn](https://yarnpkg.com/) (another package manager for nodejs)
  - [Cargo](https://github.com/rust-lang/cargo) (The [Rust](https://www.rust-lang.org/) Package manager)
  - [Systemd](https://systemd.io/) (Init system  and service manager for [GNU/Linux](https://www.gnu.org/gnu/linux-and-gnu.en.html))
  - [Pacman](https://archlinux.org/pacman/pacman.8.html) (Package manager for Arch Linux)
  - [GNU make](https://www.gnu.org/software/make/) (a tool that controls the generation of executables and other non-source files of a program from the program's source files. )
  - [vi family](https://texteditors.org/cgi-bin/wiki.pl?ViFamily) (The VI family is a popular member of the [TextEditorFamilies](https://texteditors.org/cgi-bin/wiki.pl?TextEditorFamilies), especially on UNIX and Linux machines.)

### CODE REVIEW

File: hydras/vi [hunk 1]
```diff
@@ -0,0 +1,4 @@
+v,Vi-Family,
+vn,Neovim,nvim 
+vv,Vim,vim
+vi,Vi
```
Comment (emad-elsaid) [inline hydras/vi]: this line should have a `,` at the end and a command

**Review by emad-elsaid — COMMENTED** [review]
the files that has an empty line at the end raises an error so all files shouldn't have an empty line at the end. 
I guess hydra should handle this case. it's common for editors to add empty line at the end

File: hydras/vi (hunk 1 above)
Comment (hulxv) [reply hydras/vi]: Oh. Sorry, I missed it.

**Comment by emad-elsaid** [comment]
with the last change we're missing just one more change to remove any empty lines from the files. to make sure the files works try running `hydra hydras/*` it should show the hydra UI not throwing an error

**Comment by hulxv** [comment]
Thanks @emad-elsaid for merging, and I'm sorry for the slow response. I'm far away from my machine these days and I couldn't test the hydras as you said.

**Pull request merged**